module/applications/api/primary-sheet-mixin.mjs

```diff
@@ -503,8 +503,8 @@ export default function PrimarySheetMixin(Base) {
     static sortItemsPriority(a, b) {
       return a.system.linkedActivity?.item?.name.localeCompare(b.system.linkedActivity?.item?.name, game.i18n.lang)
         || ((a.system.level ?? 0) - (b.system.level ?? 0))
-        || a.system.preparation?.mode?.localeCompare(b.system.preparation?.mode, "en")
-        || ((b.system.preparation?.prepared ?? 0) - (a.system.preparation?.prepared ?? 0))
+        || (a.system.method ?? "").compare(b.system.method ?? "")
+        || ((a.system.prepared ?? 0) - (b.system.prepared ?? 0))
```

Comment:
So I reversed these when fixing a different bug recently because it made more sense to me that when using "Priority" mode, prepared and always prepared spells would sort above non-prepared spells.

Reply:
I guess it only matters for cantrips, since otherwise spells with different methods are never mixed together in the same section. But I agree you'd want to see your prepared cantrips first I'd imagine.

Reply:
Aw, wasn't quite clear about this. I rather than moving the method after prepared, I was thinking of reversing the sorting order of prepared itself.

```suggestion
        || ((b.system.prepared ?? 0) - (a.system.prepared ?? 0))
```

Current order:
<img width="540" height="397" alt="Screenshot 2025-07-15 at 16 25 33" src="https://github.com/user-attachments/assets/21e35a04-6441-47ee-84de-5c8ce5090677" />

Adjusted order:
<img width="543" height="387" alt="Screenshot 2025-07-15 at 16 25 43" src="https://github.com/user-attachments/assets/9e555ca4-6b00-4a59-8bdc-fa21e49ab98e" />